services/opaque.go

```diff
@@ -21,10 +21,6 @@ const (
 	opaqueSecretKeyEnv  = "OPAQUE_SECRET_KEY"
 	opaquePublicKeyEnv  = "OPAQUE_PUBLIC_KEY"
 	opaqueFakeRecordEnv = "OPAQUE_FAKE_RECORD"
-
-	opaqueArgon2TimeParam     = 2
-	opaqueArgon2ParallelParam = 1
-	opaqueArgon2MemoryParam   = 19456
 )
 
 var (
```

Comment:
Should the variable on the next line be dropped?

Reply:
good catch, updated